### DISCUSSION TITLE
Desktop app for self hosted installation

### DISCUSSION THREAD

**Discussion by LudovicOpenium** [outline/outline#6209] [Self Hosting] [answered]
Hi,

Is there a way to use MacOS desktop app for self hosted installation ? (Not the PWA app)
If so, what is the way to go ?
Is the app code in a specific private repo ? Just to know if we can do the changes ourself.

Best regards,

**Comment by tommoor** [comment, marked as answer]
The desktop apps are only available for cloud-hosted customers at this time I'm afraid.

**Reply by LudovicGodart** [reply]
Thank you for this clarification 👍
